src/domain/Popovers/DropdownMenu/subcomponents/Section.tsx

```diff
@@ -102,20 +111,84 @@ class Section extends React.PureComponent<ISectionProps, never> {
 
     const Component = this.component
     return (
-      <StyledSection {...this.styleProps}>
-        <Component
-          {...componentProps}
-          onClick={onClick}
-          href={href}
-        >
-          {leftComponent && <span className='left-component'>{leftComponent}</span>}
-          {text}
-          {rightComponent && <span className='right-component'>{rightComponent}</span>}
-        </Component>
-      </StyledSection>
+      tooltipMessage ? (
+          <StyledSection {...this.styleProps}>
+            <Component
+              {...componentProps}
+              onClick={onClick}
+              href={href}
+            >
+              <StyledSectionContent>
```

Comment:
Sorry I meant moving the condition down here so you don't have to repeat this section twice
```
StyledSection
Component
```